client/client-multi/client-c/Modules/cppcheck/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2014-2016 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This cmake list is resposible for cppcheck.
+
+add_custom_target(cppcheck
+    COMMAND cppcheck --quiet --enable=all --std=c99	--suppress=unusedFunction --force
```

Comment:
Too many spaces after `--std=c99`. In general, it would be good to preserve original spacing:
```
COMMAND cppcheck --quiet --enable=all --std=c99 --suppress=unusedFunction
                 --force --error-exitcode=1 --template=gcc -I src/kaa
                 --inline-suppr
                 src/ test/
```